### PR TITLE
add python3-ubi image

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
Ready

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
add python3-rhel image which will be a base image for python3 that is based on redhat linux

### CODE REVIEW

File: docker/python3-rhel/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# python3.10.8 at the moment - if updated please update this comment
+ARG BASE_UBI_TAG=8.6
+
+FROM registry.access.redhat.com/ubi8:${BASE_UBI_TAG} as build
```
Comment (JudahSchwartz) [inline docker/python3-rhel/Dockerfile:4]: would put ubi8 as var also

File: docker/python3-ubi/verify.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import warnings
```
Comment (glicht) [inline docker/python3-ubi/verify.py:1]: This file should be a symbolic link to the python3/verify.py file as we do in python3-deb. See for example: https://github.com/demisto/dockerfiles/blob/master/docker/python3-deb/verify.py

File: docker/python3-ubi/Pipfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+[packages]
+requests = {extras = ["socks"],version = "*"}
+olefile = "*"
+python-dateutil = "*"
+dateparser = "*"
+pyyaml = "*"
+funcy = "*"
+tldextract = "*"
+pip = "*"
```
Comment (glicht) [inline docker/python3-ubi/Pipfile:16]: Why is this file different from the Pipfile used by python3? 

See: https://github.com/demisto/dockerfiles/blob/master/docker/python3/Pipfile

**Review by glicht — CHANGES_REQUESTED** [review]

File: docker/python3-ubi/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,84 @@
+# python3.10.8 at the moment - if updated please update this comment
+ARG BASE_UBI_TAG=8.6
```
Comment (glicht) [inline docker/python3-ubi/Dockerfile:2]: Why not use 8.7?

Comment (GuyAfik) [reply docker/python3-ubi/Dockerfile]: updated to 8.7

File: docker/python3-ubi/verify.py (hunk 2 above)
Comment (GuyAfik) [reply docker/python3-ubi/verify.py]: done

File: docker/python3-ubi/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    echo '/usr/local/lib' > /etc/ld.so.conf && \
+    ldconfig
+
+COPY requirements.txt .
+RUN pip3.10 install -r requirements.txt
+
+FROM registry.access.redhat.com/${BASE_UBI_VERSION}:${BASE_UBI_TAG}
+
+RUN dnf update -y --nodocs && \
+    dnf clean all && \
```
Comment (glicht) [inline docker/python3-ubi/Dockerfile:51]: Let's also add procps-ng so we have `ps` available for troubleshooting. You can add:
```
dnf install --nodocs -y  procps-ng
```

Comment (GuyAfik) [reply docker/python3-ubi/Dockerfile:52]: done

File: docker/python3-ubi/Pipfile (hunk 3 above)
Comment (GuyAfik) [reply docker/python3-ubi/Pipfile]: done

**Review by glicht — APPROVED** [review]
Looks good

**Pull request merged**